charon/src/bin/charon-driver/translate/translate_constants.rs

```diff
@@ -145,9 +145,28 @@ impl<'tcx, 'ctx, 'ctx1> BodyTransCtx<'tcx, 'ctx, 'ctx1> {
                 let be = self.translate_constant_expr_to_constant_expr(span, be)?;
                 RawConstantExpr::Ref(Box::new(be))
             }
-            ConstantExprKind::MutPtr(be) => {
-                let be = self.translate_constant_expr_to_constant_expr(span, be)?;
-                RawConstantExpr::MutPtr(Box::new(be))
+            ConstantExprKind::Cast { .. } => {
+                error_or_panic!(
+                    self,
+                    span,
+                    &format!("Unsupported constant: `ConstantExprKind::Cast {{..}}`",)
+                )
+            }
+            ConstantExprKind::RawBorrow {
+                mutability: true, ..
+            } => {
+                error_or_panic!(
+                    self,
+                    span,
+                    &format!("Unsupported constant: `ConstantExprKind::RawBorrow {{mutability: true, ..}}`",)
+                )
+            }
+            ConstantExprKind::RawBorrow {
+                mutability: false,
```

Comment:
Should be `true` :D

Reply:
oops, I pushed an embarrassing commit fixing this embarrassing mistake, thanks